.github/workflows/aks.yml

```diff
@@ -35,7 +37,34 @@ env:
   AKS_MACHINE_TYPE: 'Standard_D3_v2'
 
 jobs:
+  linter:
+    runs-on: self-hosted
```

Comment:
I wonder why we are using self-hosted runners for that job. I expected to see ubuntu with github runners.
Maybe to avoid limitations on free github actions?

Reply:
Maybe yes, we can check this outside of this PR anyway.